contracts/access/AccessController.sol

```diff
@@ -54,16 +55,16 @@ contract AccessController is IAccessController, GovernableUpgradeable, UUPSUpgra
     }
 
     /// @notice initializer for this implementation contract
-    /// @param governance The address of the governance contract
-    function initialize(address governance) external initializer {
-        __GovernableUpgradeable_init(governance);
+    /// @param accessManaged The address of the governance contract
+    function initialize(address accessManaged) external initializer {
```

Comment:
AccessManager or AccessManaged? Need to be consistent

Reply:
0885c24